packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -545,11 +561,25 @@ class _RailDestination extends StatelessWidget {
     final Widget content;
     switch (labelType) {
       case NavigationRailLabelType.none:
+        final Widget iconWithIndicator =  Stack(
+          alignment: Alignment.center,
+          children: <Widget>[
+            if (useIndicator)
+              NavigationIndicator(
+                animation: destinationAnimation,
+                height: 56,
+                width: 56,
+                borderRadius: 100,
```

Comment:
Nit: Maybe pull the `56` out into a const and use `56/2` as the borderRadius to make it more obvious that its a circle

Reply:
Done.